src/quo/components/text_combinations/page_top/component_spec.cljs

```diff
@@ -0,0 +1,122 @@
+(ns quo.components.text-combinations.page-top.component-spec
+  (:require [quo.components.text-combinations.page-top.view :as page-top]
+            [test-helpers.component :as h]))
+
+(defonce mock-picture {:uri (js/require "../resources/images/mock2/user_picture_male4.png")})
+
+(def context-tag-data
+  {:type                :community
+   :state               :default
+   :customization-color :army
+   :community-logo      mock-picture
+   :community-name      "Coinbase"
+   :emoji               "😝"})
+
+(h/describe "Page Top"
```

Comment:
Could you please also add the test rendering component without any props?

Reply:
Hi @vkjr 
I'm wondering why do we want to sucessfully render the component without props?